docs/pages/access-controls/access-request-plugins/ssh-approval-pagerduty.mdx

```diff
@@ -1,76 +1,419 @@
 ---
-title: SSH login approval via Pager Duty
-description: How to configure SSH login approval using PagerDuty and Teleport
-h1: SSH Login Approval using PagerDuty
-videoBanner: RzTB4Lsm09M
+title: Access Requests with PagerDuty
+description: How to set up Teleport's PagerDuty plugin for privilege elevation approvals.
 ---
 
-This guide will talk through how to setup Teleport with PagerDuty. Teleport's
-PagerDuty integration allows you to treat Teleport access and permission
-requests as PagerDuty incidents, notify the appropriate team, and approve
-or deny the requests via PagerDuty special action.
+With Teleport's PagerDuty integration, engineers can access the infrastructure
+they need to resolve incidents quickly—without longstanding admin permissions
+that can become a vector for attacks.
 
-## Setup
+Teleport's PagerDuty integration allows you to treat Teleport Role Access Requests as
+PagerDuty incidents, notify the appropriate on-call team, and approve or deny
+the requests via PagerDuty. You can also configure the plugin to approve Role
+Access Requests automatically if the user making the request is on the on-call
+team for a service affected by an incident.
 
-### Prerequisites
+This guide will explain how to set up Teleport's Access Request plugin for
+PagerDuty.
 
-This guide assumes that you have:
+<video controls>
+  <source
+    src="../../../img/enterprise/plugins/pagerduty/pagerduty-demo.mp4"
+    type="video/mp4"
+  />
+
+  <source
+    src="../../../img/enterprise/plugins/pagerduty/pagerduty-demo.webm"
+    type="video/webm"
+  />
 
-- A running Teleport Cluster
-- Admin privileges with access to `tctl`
-- A PagerDuty account, with access to creating a new API token
-- A node to run the plugin. We recommend running it alongside the Teleport Proxy for convenience
+  Your browser does not support the video tag.
+</video>
 
-Teleport Cloud requires that plugins connect through the Proxy Service (`mytenant.teleport.sh:443`). Open Source and Enterprise installations can connect to the Auth Service (`auth.example.com:3025`) directly.
+## Prerequisites
```

Comment:
Should self-hosted prerequisites also mention a Teleport cluster or you think it's pretty obvious from the context?

Reply:
I've edited `commercial-prereqs-tabs.mdx` so that both the Cloud and Enterprise tabs mention a Teleport cluster.